.github/workflows/ci.yml

```diff
@@ -51,12 +47,55 @@ jobs:
         run: composer validate
 
       - name: Composer version
-        run: composer --version
+        run: |
+          composer --version
+          composer install
+
+      - name: Run PHPStan
+        run: composer stan
+
+      - name: Run CodeStyle checks
+        run: composer cs-check
+
+  validation-74:
```

Comment:
lowest doesnt mean you can stop testing 7.4 with highest.

Reply:
it doesn't work with highes

Reply:
It doesnt need to. The definition of it is: Lowest with lowest.